packages/os/os.spec

```
@@ -169,8 +186,25 @@ Requires: %{_cross_os}settings-defaults
 %package -n %{_cross_os}migration
 Summary: Tools to migrate version formats
 Conflicts: %{_cross_os}image-feature(no-in-place-updates)
+Requires: %{_cross_os}migration(binaries)
 %description -n %{_cross_os}migration
 
+%package -n %{_cross_os}migration-bin
+Summary: Binaries to migrate version formats
+Provides: %{_cross_os}migration(binaries)
+Requires: (%{_cross_os}image-feature(no-fips) and %{_cross_os}migration)
+Conflicts: (%{_cross_os}image-feature(no-inplace-updates) or %{_cross_os}image-feature(fips) or %{_cross_os}migration-fips-bin)
```

```suggestion
Conflicts: (%{_cross_os}image-feature(no-in-place-updates) or %{_cross_os}image-feature(fips) or %{_cross_os}migration-fips-bin)
```

This could also just be a separate `Conflicts`.